apps/delivery-options/src/views/MyParcelDeliveryOptions/Pickup/PickupLocationList/PickupLocationCarrierGroup.vue

```diff
@@ -47,15 +47,18 @@ import {
   type SelectOption,
   SHOW_MORE_LOCATIONS,
   useLoadMore,
+  type CarrierIdentifier,
+  resolveCarrierName,
 } from '@myparcel-do/shared';
-import {type CarrierName, DeliveryTypeName} from '@myparcel/constants';
+import {DeliveryTypeName} from '@myparcel/constants';
 import {getDeliveryTypePrice} from '../../../../utils';
 import {useLanguage, useSelectedPickupLocation} from '../../../../composables';
 import {GroupInput, DoButton, PriceTag} from '../../../../components';
 import PickupLocationListItem from './PickupLocationListItem.vue';
 import PickupLocationDetails from './PickupLocationDetails.vue';
 
-const props = defineProps<{carrier: CarrierName; options: SelectOption<T>[]}>();
+const props = defineProps<{carrier: CarrierIdentifier; options: SelectOption<T>[]}>();
+const carrierName = resolveCarrierName(props.carrier);
```

Comment:
Dit is niet reactive, moet in een computed property

Reply:
Gedaan